### PR TITLE
Himanshu/20 06 2022

### PR DESCRIPTION
**Pull request by unknownhad**
Updating date of release.

### CODE REVIEW

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -25,14 +25,24 @@ layout: list
   <tbody>
     <tr>
       <td>2022-06-20</td>
-      <td>2022-06-27</td>
+      <td>2022-06-05</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>2022-07-05</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -25,14 +25,24 @@ layout: list
   <tbody>
     <tr>
       <td>2022-06-20</td>
-      <td>2022-06-27</td>
+      <td>2022-07-05</td>
       <td>Cloudflare Specials</td>
       <td>...34780914</td>
       <td>100532</td>
       <td>Vulnerabilty scanner activity</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md:32]:
```suggestion
      <td>Vulnerability scanner activity</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 3]
```diff
@@ -25,14 +25,24 @@ layout: list
...
       <td>N/A</td>
       <td>Block</td>
     </tr>
+    <tr>
+      <td>2022-06-20</td>
+      <td>2022-07-05</td>
+      <td>Cloudflare Specials</td>
+      <td>...N/A</td>
+      <td>100532</td>
+      <td>Vulnerabilty scanner activity</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md:42]:
```suggestion
      <td>Vulnerability scanner activity</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 4]
```diff
@@ -25,14 +25,24 @@ layout: list
...
-      <td>2022-06-27</td>
+      <td>2022-07-05</td>
       <td>Cloudflare Specials</td>
       <td>...34780914</td>
       <td>100532</td>
       <td>Vulnerabilty scanner activity</td>
       <td>N/A</td>
       <td>Block</td>
     </tr>
+    <tr>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md:36]: The two rows are very similar (both related to `100532`). Should it be just one table row here?

Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md:36]: Yes, it's a copy paste error.

**Pull request closed without merging**